build.gradle

```diff
@@ -35,12 +39,18 @@ subprojects {
         jUnitVersion = "5.+"
         jUnitLegacyVersion = "4.+"
         mockitoVersion = "4.+"
-        slf4jVersion = "1.7.+"
+        slf4jVersion = "2.+"
         spectatorVersion = "1.+"
-        springVersion = "5.+"
+        springVersion = "6.+"
```

Comment:
Given that we need to keep the bytecode on Java 8, and Spring is only used for testing, I think it makes sense to stay on Spring 5. Is there a compelling reason to move to Spring 6?